QUARKUS-6471.md

```diff
@@ -0,0 +1,41 @@
+# QUARKUS-6471 - Support for OAuth2 Protected Resource Metadata (RFC 9728)
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6471
+
+Upstream PRs: 
+
+- https://github.com/quarkusio/quarkus/pull/48734
+
+This PR adds a partial support for https://datatracker.ietf.org/doc/rfc9728/. It allows Quarkus app to expose URL of authorization server it uses.
+
+## Scope of the testing
+
+Check, that the URL is exposed with `quarkus.oidc.resource-metadata.enabled` property set to true and is not exposed otherwise.
+Check, that `quarkus.oidc.resource-metadata.resource` can be used to specify the protected resource. 
+
+## Existing test coverage
+
+There is some tests upstream, but they do not include running the real OIDC provider or changing `quarkus.oidc.resource-metadata.resource` property.
```

Comment:
`integration-tests/oidc-code-flow/src/test/java/io/quarkus/it/keycloak/CodeFlowTest.java` runs against Keycloak and sets `quarkus.oidc.tenant-nonce.resource-metadata.resource` which then tests.

Reply:
Good catch! Changed the wording.